packages/lesswrong/components/recentDiscussion/RecentDiscussionSubscribeReminder.tsx

```diff
@@ -17,16 +19,12 @@ const styles = (theme: ThemeType): JssStyles => ({
   root: {
     marginBottom: theme.spacing.unit*4,
     position: "relative",
-    minHeight: 58,
     backgroundColor: "rgba(253,253,253)",
-    
+
     padding: 16,
     ...theme.typography.body2,
-    
-    border: "1px solid #aaa",
-    borderRadius: 10,
-    boxShadow: "5px 5px 5px rgba(0,0,0,20%)",
-    
+    boxShadow: theme.boxShadow,
```

Comment:
I considered getting rid of the root box/capsule entirely to make this component leaner, but the main constraint was that the LessWrong signup form is relatively large and feels like it needs the container

Reply:
I'm not sure I see your vision. Maybe describe it to me live, or maybe 🚢  it

Reply:
Something like this:

<img width="821" alt="Screen Shot 2021-09-09 at 11 32 22 PM" src="https://user-images.githubusercontent.com/9057804/132771217-8633a0ca-13a6-4155-a05f-1e4be10f5ccf.png">

Reply:
I like how this looks.